### PR TITLE
Update customer-key-set-up.md

### PR DESCRIPTION
**Pull request by brdesai**
Updated licensing information based on SKU changes from Marketing. The changes to go live on April 1, 2020.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/customer-key-set-up.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md]:
```suggestion
Before you get started, ensure that you have the appropriate licensing for your organization. Starting April 1, 2020, Customer Key in Office 365 is offered in M365 E5 or E5 Compliance SKU. Office 365 Advanced Compliance SKU is no longer available for procuring new licenses. Customers with existing Office 365 Advanced Compliance licenses will continue to be supported.  
```

File: microsoft-365/compliance/customer-key-set-up.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md]:
```suggestion
To understand the concepts and procedures in this topic, review the [Azure Key Vault](https://docs.microsoft.com/azure/key-vault/) documentation. Also, become familiar with the terms used in Azure, for example, [Azure AD tenant](https://docs.microsoft.com/previous-versions/azure/azure-services/jj573650(v=azure.100)#what-is-an-azure-ad-tenant).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@brdesai  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by brdesai** [comment]
Suggested changes have been committed.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by brdesai** [comment]
@KCCross, @JohanFreelancer9  Do you want me to close the pull request or are you going to do it.

**Comment by brdesai** [comment]
@KCCross, @JohanFreelancer9  I'd to make one more commit to update the list of supported SKUs based on the discussion with PMM.

**Review by shilpabothra — COMMENTED** [review]
Should we add something along the lines of: 
We will stop selling the Office 365 Advanced Compliance as a standalone subscription. When current subscriptions expire, customers should transition to one of the subscriptions above. We are not discontinuing any of the existing compliance features.

**Review by KCCross — CHANGES_REQUESTED** [review]
We are speaking to customers in this content so we do not say customers :) So, modify the text starting with "Starting April 1" and ending with "to be supported" as follows:

Starting April 1, 2020, Customer Key in Office 365 is offered in Office 365 E5, M365 E5, M365 E5 Compliance and M365 E5 Information Protection & Governance SKUs. Office 365 Advanced Compliance SKU is no longer available for procuring new licenses. Existing Office 365 Advanced Compliance licenses will continue to be supported.

File: microsoft-365/compliance/customer-key-set-up.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-set-up.md]:
```suggestion
Before you get started, ensure that you have the appropriate licensing for your organization. Starting April 1, 2020, Customer Key in Office 365 is offered in Office 365 E5, M365 E5, M365 E5 Compliance, and M365 E5 Information Protection & Governance SKUs. Office 365 Advanced Compliance SKU is no longer available for procuring new licenses. Customers with existing Office 365 Advanced Compliance licenses will continue to be supported.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit. Please add a comma as indicated.

**Comment by brdesai** [comment]
@KCCross , @JohanFreelancer9 The changes have been made, take a look and do the needful to merge the changes. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review.

Thank you.

cc: @kenwith

**Comment by KCCross** [comment]
I'm concerned at how these did not make it live after the PR was merged. Please verify the accuracy against the new service description release in June.

**Pull request merged**
